Dockerfile-ultra-lite

```diff
@@ -1,5 +1,5 @@
-# Build jbig2enc in a separate stage
-FROM bellsoft/liberica-openjdk-alpine:17
+# use alpine
+FROM alpine:latest
```

Comment:
I preffer pinning version, but it seems this repo isn't manged by dependabot/renovate, etc. right?

Reply:
dependabot is running, but i believe its only reviewing the java gradle versions not docker, let me see if i can get it for both

Reply:
Added for docker https://github.com/Stirling-Tools/Stirling-PDF/commit/ee96d2a0e3b79c5bd577c5b6f33d329cb42522a5

Reply:
Can you pin version, would be best

Reply:
are you sure that dependabot also detect files which are not named Dockerfile? I only know it from renovate, renovate only dedects Dockerfiles when theier filename ends with Dockerfile (so it dedects Dockerfile, abc.Dockerfile, but not Dockerfile-abc), not sure how dependabot handels this

Reply:
https://github.com/Stirling-Tools/Stirling-PDF/pull/635/files
seems so